clients/cli/src/print_tree.rs

```diff
@@ -0,0 +1,30 @@
+use crate::error::CliResult;
```

Comment:
CI seems to claim that your code doesn't compile. Does this work for you locally, or is this just pseudocode?